### ISSUE TITLE
Kube test network : Pods can not resolve external DNS host names 

### ISSUE THREAD

**Issue by jkneubuh** [hyperledger/fabric-samples#550]
Pods running in the Kubernetes Test Network are no longer able to resolve external DNS entries.  E.g.: 

Setup: 
- Set up a new Linux host 
- git clone https://github.com/hyperledger/fabric-samples.git
- cd fabric-samples/test-network-k8s 
- `./network kind` 

Resolve a local DNS host / pod (OK): 
```
root@hexapody1:~# kubectl apply -f https://k8s.io/examples/admin/dns/dnsutils.yaml
pod/dnsutils created
root@hexapody1:~# kubectl exec -i -t dnsutils -- nslookup kubernetes.default
Server:		10.96.0.10
Address:	10.96.0.10#53

Name:	kubernetes.default.svc.cluster.local
Address: 10.96.0.1
```

Resolve an external DNS host name (FAILS): 
```
root@hexapody1:~# kubectl exec -i -t dnsutils -- nslookup www.ibm.com 
Server:		10.96.0.10
Address:	10.96.0.10#53

** server can't find www.ibm.com.fyre.ibm.com: SERVFAIL

command terminated with exit code 1
```

**Comment by jkneubuh** [comment]
This is caused by [KIND Issue #2326](https://github.com/kubernetes-sigs/kind/issues/2326). 

Recently we enabled the [`IPVS` proxy routing](https://kubernetes.io/docs/concepts/services-networking/service/#proxy-mode-ipvs) feature for the Kubernetes Test Network, in part to expose additional options for load balancing a set of gateway peers behind a shared service / DNS alias.  Unfortunately this tickled a bug in the way KIND proxies DNS lookups through the Docker network, causing external lookups to fail.

Fix this issue by : 

1. revert IPVS -> IPTABLES routing when bootstrapping the KIND cluster.
2. Update the Fabric Deployment Guide with additional instructions for IPVS Service Routing.
3. Document the [`route_localnet=1`](https://github.com/kubernetes-sigs/kind/issues/2326#issuecomment-881969982) workaround, for KIND environments validating IPVS: 

```
# Workaround https://github.com/kubernetes-sigs/kind/issues/2326
for node in $(kind get nodes); do
  docker exec "$node" sysctl net.ipv4.conf.all.route_localnet=1
done
```

I've confirmed the above workaround re-enables DNS. 

@denyeart @mbwhite cc:

**Comment by jkneubuh** [comment]
Fixed in PR #729
